source/index.rst

```diff
@@ -1,22 +1,30 @@
 .. Cloudant Python SDK documentation master file, created by
    sphinx-quickstart on Tue Jan  5 13:47:50 2021.
-   Copyright IBM Corporation 2021.
+   Copyright IBM Corporation 2021, 2023.
    SPDX-License-Identifier: Apache-2.0
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Cloudant Python SDK's documentation!
-===============================================
+Cloudant Python SDK's Documentation
+===================================
+
+CloudantV1 class
+----------------
+
+:class:`CloudantV1 <ibmcloudant.cloudant_v1.CloudantV1>`
```

Comment:
Really good idea to link this here.
Maybe in some later follow-on work we can figure out a way to add the available functions that correspond to API operations to the TOC or something.

Reply:
Yes, that'd be nice to have and it's trivial to add manual `cloudantv1.rst` with whatever content we want, but automatically getting to `toc` only section of another file is not obvious. Brute force hack would be to cut a middle out of generated `cloudant_v1.py` into a separate file and generate index from it, though I'm concerned what that'd do with links, so I didn't pursue that idea.